src/hotspot/share/code/nmethod.cpp

```diff
@@ -1219,29 +1223,29 @@ void nmethod::verify_clean_inline_caches() {
   RelocIterator iter(this, oops_reloc_begin());
   while(iter.next()) {
     switch(iter.type()) {
-      case relocInfo::virtual_call_type:
-      case relocInfo::opt_virtual_call_type: {
+      case relocInfo::virtual_call_type: {
         CompiledIC *ic = CompiledIC_at(&iter);
-        CodeBlob *cb = CodeCache::find_blob(ic->ic_destination());
+        CodeBlob *cb = CodeCache::find_blob(ic->destination());
         assert(cb != nullptr, "destination not in CodeBlob?");
         nmethod* nm = cb->as_nmethod_or_null();
         if( nm != nullptr ) {
```

Comment:
Maybe fix this while in the area similar to other places.

Reply:
Sorry, I'm not sure I understand what you would like me to change here.

Reply:
```suggestion
        if (nm != nullptr ) {
```
Sorry.